### PR TITLE
Timeout enlarge

### PR DESCRIPTION
**Pull request by bodqhrohro**
With 60 seconds timeout some tests fail to finish in time on my weak laptop, so I suggest to increase it, if that wouldn't have any negative impact on CI or whatever.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@
 .vscode
 .coverage*
 .idea
+/signingkeyserver.db
```
Comment (anoadragon453) [inline .gitignore:16]: Mmm, what generates this file? Also ideally this change would be in a separate PR :slightly_smiling_face:

File: run-tests.pl [hunk 2]
```diff
@@ -758,7 +758,7 @@ sub _run_test0
 
       Future->wait_any(
          $f_setup,
-         $loop->delay_future( after => 60 )
+         $loop->delay_future( after => 300 )
```
Comment (anoadragon453) [inline run-tests.pl:761]: 300 seconds is a bit of a jump from 60. Perhaps we could start with x2 = 120?

What is the maximum amount of time a test takes on your machine?

**Review by anoadragon453 — COMMENTED** [review]
Hello, and thank you for your PR :slightly_smiling_face: 

Could you tell us whether there is a particular test that's timing out, or are you having trouble with lots of them?

Also, an alternative solution (though a more complex one) would be to just make the test timeout configurable.

**Comment by anoadragon453** [comment]
Hmm, not entirely sure where the CI failures are coming from. The error line is:

```
# fixture failed - TODO: Missing fixture dependency can_register_with_secret
```

You may need to merge latest develop to your branch.

**Comment by bodqhrohro** [comment]
Well, after some investigation, I find this PR useless. There are lots of other timeouts in the code.

Instead it should probably be implemented other way: as an environment variable specifying a factor that **all** hardcoded timeouts should be multiplied to. This way, only those having a weak machine will have to tune it, in a much more flexible way.

Should I close it for now and post an issue about the new approach, until me or someone else implements it?

File: .gitignore (hunk 1 above)
Comment (bodqhrohro) [reply .gitignore:16]: Dendrite does.

Does it happen because this file is not yet overridden in `Dendrite.pm` to Postgres like other files are? Should `*.db` be in `.gitignore` instead for the case when the tests are run without the `POSTGRES` variable?

File: run-tests.pl (hunk 2 above)
Comment (bodqhrohro) [reply run-tests.pl:761]: How do I measure it? TAP doesn't seem to print start/end time for successful tests by default.

Actually, there's no upper limit, as the machine is loaded with other tasks and may easily go into the thrashing state.

Comment (bodqhrohro) [reply run-tests.pl:761]: Experiments with #996 show that with ×2 scaling, there still are 73 tests timing out; with ×3 and more, there still remain about 22–24 tests timing out, which all seem to be expected to fail because of timeouts though.

**Comment by bodqhrohro** [comment]
Replaced by #996.
